netweaver/monitoring.sls

```diff
@@ -5,22 +5,30 @@ prometheus_sap_host_exporter_pkg:
   pkg.installed:
     - name: prometheus-sap_host_exporter
 
-sap_host_exporter_exporter_service:
+# the sid, instance number pair of a node is unique, so we need to adapt configuration
+{% for node in netweaver.nodes if host == node.host and node.sap_instance != "db" %}
+{% set instance = node.instance %}
```

Comment:
Still this first line `{% set instance = node.instance %}` must be removed, it doesn't do anything